### PR TITLE
Update and completion of Logic Node documentation

### PR DESCRIPTION
**Pull request by xavco63**
I have updated and completed some part of the logic nodes section.

@IzaZed will merge it I will inform him on Discord no worry 😁

### CODE REVIEW

File: source/manual/logic_nodes/data/list/append.rst [hunk 1]
```diff
@@ -28,7 +30,10 @@ Done
    *True* if node performs successfully, else *False*.
 
 List
-   Resulting list.
+   The given list with the new value attached at the end.
 
 .. note::
    Square socket indicates a list.
+
+.. note::
```
Comment (IzaZed) [inline source/manual/logic_nodes/data/list/append.rst:38]: Is this note really necessary?

File: source/manual/logic_nodes/data/list/list_from_items.rst [hunk 2]
```diff
@@ -9,23 +9,27 @@
...
 
 Outputs
 ++++++++++++++++++++++++++++++
 
 List
-   Resulting populated list with added items.
+   New list with the added items.
+
+.. note::
+   Leaving a *Item* input blank will result in a Null item in your list.
```
Comment (IzaZed) [inline source/manual/logic_nodes/data/list/list_from_items.rst:34]: typo: "Leaving an item ...."

File: source/manual/logic_nodes/data/list/remove_value.rst [hunk 3]
```diff
@@ -9,14 +9,16 @@
 Remove Value
 ==============================
 
+Remove the first itteration of a given value in a list.
```
Comment (IzaZed) [inline source/manual/logic_nodes/data/list/remove_value.rst:12]: typo: "iteration"

File: source/manual/logic_nodes/data/list/remove_value.rst [hunk 4]
```diff
@@ -29,3 +31,8 @@ Done
 
 List
    Resulting list after value removal.
+
+.. note::
+   This node will only remove the first itteration in the list,
```
Comment (IzaZed) [inline source/manual/logic_nodes/data/list/remove_value.rst:36]: typo: "iteration"

File: source/manual/logic_nodes/time/time_data.rst [hunk 5]
```diff
@@ -9,14 +9,22 @@
 Time Data
 ==============================
 
+Return usefull datas about the time within your game.
```
Comment (IzaZed) [inline source/manual/logic_nodes/time/time_data.rst:12]: typo: "useful data"

File: source/manual/logic_nodes/time/time_data.rst [hunk 6]
```diff
@@ -9,14 +9,22 @@
...
 Outputs
 ++++++++++++++++++++++++++++++
 
 Time
-   todo
+   The number of seconds elapsed since the game engine was launched (Float).
+
+   .. note::
+      The value returned here is independent of a scene change (time will continue to be counted without resetting the count),
+      but it will be reset after loading a .blend file because it relaunch the game engine.
```
Comment (IzaZed) [inline source/manual/logic_nodes/time/time_data.rst:22]: typo: "... because it relaunches the ...."

**Review by IzaZed — COMMENTED** [review]
A few typos, otherwise this is brilliant! Thank you!

File: source/manual/logic_nodes/data/list/append.rst (hunk 1 above)
Comment (xavco63) [reply source/manual/logic_nodes/data/list/append.rst:38]: Yes I think for people not coming from programming and don't know the behavior of append. But if you really want to remove it, i'll!

**Pull request merged**